subsystem/cluster_resource_rosa_test.go

```diff
@@ -757,6 +766,7 @@ var _ = Describe("Cluster creation", func() {
 				"label_key2" = "label_value2"
 			}
 			sts = {
+				oidc_config = {},
```

Comment:
Not sure if there's a correct way to do this so that adding this struct is not needed